site/content/en/docs/Guides/Client SDKs/local.md

```diff
@@ -207,6 +207,18 @@ go run cmd/sdk-server/main.go --local
 
 Commandline flags (e.g. `--local`) are exactly the same as command line flags when utilising a pre-built binary.
 
+**Enabling Specific Features:**
+For development and testing purposes, you might want to enable specific features in the SDK Server. To do this, set the FEATURE_GATES environment variable before running the server. For instance:
+```bash
+FEATURE_GATES="FeatureName=true" go run cmd/sdk-server/main.go --local
+```
+
+Replace `FeatureName` with the name of the feature you want to enable. This command starts the local SDK Server with the specified feature activated.
+
+**Setting the HTTP port:**
+To customize the SDK Server, set the HTTP port to 9358, as used by the local server, by running this command in your terminal: `export AGONES_SDK_HTTP_PORT=9358`
```

Comment:
```suggestion
```

This isn't accurate for the local SDK server, it's either --http-port argument, or HTTP_PORT (similar for grpc). Since port changing isn't that common, I'd suggest not worrying about documenting this -- unless anyone feels strongly otherwise?

If people really want it, they can run `--help` on the local server as well to get all the command line params.

Reply:
I've eliminated this `Setting the HTTP port:` section